ingestion-beam/src/main/java/com/mozilla/telemetry/contextualservices/ContextualServicesReporterOptions.java

```diff
@@ -30,6 +30,12 @@
 
   void setReportingEnabled(ValueProvider<Boolean> value);
 
+  @Description("Size (in minutes) of the window for aggregation of reporting requests.")
```

Comment:
I'd like to use a Duration here rather than assuming minutes. See this pattern in https://github.com/mozilla/gcp-ingestion/blob/e023a6a4c740633ce64ca726a59ed95e034ddb96/ingestion-beam/src/main/java/com/mozilla/telemetry/options/SinkOptions.java#L226-L232